odata_query/django/django_q.py

```diff
@@ -50,57 +47,48 @@ def visit_Null(self, node: ast.Null) -> str:
 
     def visit_Integer(self, node: ast.Integer) -> Value:
```

Comment:
These visit methods now look mostly the same. Could they be collapsed into a single `visit__Literal`, or do you expect special cases in the future?

Reply:
Fair point. The base doesn't account for inheritance at the moment, and`List` is still a non-standard `Literal`. So with some work this can be collapsed and deduplicated, but let's keep that for a next release!